bundle/CMakeLists.txt

```diff
@@ -39,7 +39,7 @@ include(DawnCloneRepository)
 
 set(DAWN_YODA_GIT_URL "https:///github.com/Meteoswiss-APN/yoda.git"
     CACHE PATH "URL of the dawn git repository to clone")
-set(DAWN_YODA_GIT_BRANCH "1.0.0" CACHE STRING "Branch of the dawn git repository to clone")
+set(DAWN_YODA_GIT_BRANCH "1.0.1" CACHE STRING "Branch of the dawn git repository to clone")

```

Comment:
can you show me what this does?

Reply:
it added the `EXPORT_GROUP` so that  that put all the libraries under the same target group (easier to carry dependencies)